lib/OpenQA/BuildResults.pm

```diff
@@ -65,7 +65,7 @@ sub add_review_badge {
     my ($build_res) = @_;
 
     $build_res->{reviewed_all_passed} = $build_res->{passed} == $build_res->{total};
-    $build_res->{reviewed} = $build_res->{failed} > 0 && $build_res->{labeled} == $build_res->{failed};
+    $build_res->{reviewed} = $build_res->{failed} > 0 && $build_res->{labeled} >= $build_res->{failed};
```

Comment:
2 softfailed 1 failed - both softfails are labled.

Can you please write a test case that you don't get a badge? Thanks.